packages/editor/src/components/editor-interface/index.js

```diff
@@ -214,7 +215,7 @@ export default function EditorInterface( {
 				)
 			}
 			actions={
-				! isPreviewMode ? (
+				customSavePanel || (
```

Comment:
I think we should keep isPreviewMode check here. I mean if you don't have customSavePanel and you're in preview mode, we shouldn't render the save panel.

Reply:
see: https://github.com/WordPress/gutenberg/pull/62863#issuecomment-2201990675